test/function.localstack.test.ts

```diff
@@ -239,32 +239,54 @@ localstackTestSuite("functionStack", (testResource, _stack, app) => {
     }
   );
 
+  // Function serialization breaks when assigning an integration/construct to a variable in the closure.
+  // TODO: what should happen here?
+  testResource.skip(
+    "Call Lambda AWS SDK put event to bus with in closure reference",
+    (parent) => {
+      const bus = new EventBus<EventBusRuleInput>(parent, "bus");
+      const func = new Function(parent, "function", async () => {
+        const busbus = bus;
+        busbus({
+          "detail-type": "anyDetail",
+          source: "anySource",
+          detail: {},
+        });
+      });
```

Comment:
Why does it fail? Seems fine to me - we're just referencing a value.

Reply:
Pulumi tries to serialize the entire construct and finds some native functions it can't serialize.

Reply:
We are going to need to fork pulumi?

Reply:
Maybe? I wanted to build up more test cases and use that to figure out what it would take to build from scratch.